src/main/java/com/airlines/dao/DBConnector.java

```diff
@@ -0,0 +1,29 @@
+package com.airlines.dao;
+
+import com.mysql.cj.jdbc.MysqlDataSource;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.ResourceBundle;
+
+public class DBConnector {
+
+    private MysqlDataSource dataSource = new MysqlDataSource();
+
+    public DBConnector() {
+        ResourceBundle appResource = ResourceBundle.getBundle("application");
```

Comment:
Java code tends to be self-explanatory.
For example:
numbers.add(evenNumber);
reads much better than:
nmbrs.add(evnNmb);

app is good shortage for application.
But try to use short names only in a case, when full name will be too long and complex.
In this case, instead of appResource I advise:
```
ResourceBundle resource
ResourceBundle properties
ResourceBundle databaseProperties

```

Reply:
fixed in